prometheus/src/cluster_metrics.rs

```diff
@@ -25,18 +39,105 @@ fn get_vote_state(bank: &Bank, vote_pubkey: &Pubkey) -> Option<ValidatorVoteInfo
     let vote_state = vote_account.vote_state();
     let vote_state = vote_state.as_ref().unwrap_or(&default_vote_state);
 
+    let identity = vote_state.node_pubkey;
+
+    let validator_info: ValidatorInfo = match get_validator_info_accounts(bank) {
+        None => ValidatorInfo {
+            name: "unknown".to_string(),
+        },
+        Some(mut info_map) => match info_map.remove(&identity) {
```

Comment:
Yeah since we use map only for that, I just used `remove`, I we were to reuse it, then clone probably makes most sense to not introduce lifetime annotations.